### ISSUE TITLE
Missing schema renaming for QueryEvent.CurrentSchema

### ISSUE THREAD

**Issue by ffffwh** [actiontech/dtle#500]
A binlog QueryEvent has a field of schema

`func handleEvent` -> `case QueryEvent`:
```go
currentSchema := string(evt.Schema)
```

It is the default schema (also called current schema) when executing the DDL. If schema is omitted in the DDL statement, default schema will be used.

The field is passed to dtle-applier `DataEvent.CurrentSchema`, and the applier will execute `use xxx`.

It is forgotten to rename it as well when there is `TableSchemaRegex` or `TableSchemaRename`.

**Comment by ffffwh** [comment]
## Case
- Put a job `db1.tb1`, renaming `db1` to `db2`
- Wait for incremental stage
- Execute sql

```sql
use db1
alter table tb1 add column ...
```

Will see error
```
mysql.applier: Ignore error: Error 1049: Unknown database 'db1'
... then applier / job error ...
Error 1046: No database selected
```

**Comment by ffffwh** [comment]
Fixed around 00c82e8d.

Also see #557.
